draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -721,7 +744,8 @@ To this end, Verifiers MUST follow the following steps (or equivalent):
        5. Check that the `typ` of the Key Binding JWT is `kb+jwt`.
        6. Check that the creation time of the Key Binding JWT, as determined by the `iat` claim, is within an acceptable window.
        7. Determine that the Key Binding JWT is bound to the current transaction and was created for this Verifier (replay protection) by validating `nonce` and `aud` claims.
-       8. Check that the Key Binding JWT is valid in all other respects, per [@!RFC7519] and [@!RFC8725].
+       8. Calculate the digest over the Presentation and verify that it matches the value of the `_sd_hash` claim in the Key Binding JWT.
```

Comment:
I think (but am not entirely sure) that "Presentation" includes the key binding jwt, but I think the hash should exclude the key binding jwt? ('Presentation' doesn't seem to be a defined term, I opened https://github.com/oauth-wg/oauth-selective-disclosure-jwt/issues/356 )

Reply:
Suggestion looks good to me.